*** /usr/local/afm/symbol.afm	Mon Mar 26 11:40:00 1990
--- symbol.afm	Tue Oct 16 09:46:37 1990
***************
*** 177,192 ****
  C 227 ; WX 790 ; N copyrightsans ; B 49 -15 739 675 ;
  C 228 ; WX 786 ; N trademarksans ; B 5 293 725 673 ;
  C 229 ; WX 713 ; N summation ; B 14 -108 695 752 ;
! C 230 ; WX 384 ; N parenlefttp ; B 40 -293 436 926 ;
! C 231 ; WX 384 ; N parenleftex ; B 40 -85 92 925 ;
! C 232 ; WX 384 ; N parenleftbt ; B 40 -293 436 926 ;
! C 233 ; WX 384 ; N bracketlefttp ; B 0 -80 341 926 ;
! C 234 ; WX 384 ; N bracketleftex ; B 0 -79 55 925 ;
! C 235 ; WX 384 ; N bracketleftbt ; B 0 -80 340 926 ;
! C 236 ; WX 494 ; N bracelefttp ; B 201 -75 439 926 ;
! C 237 ; WX 494 ; N braceleftmid ; B 14 -85 255 935 ;
! C 238 ; WX 494 ; N braceleftbt ; B 201 -70 439 926 ;
! C 239 ; WX 494 ; N braceex ; B 201 -80 255 935 ;
  C 240 ; WX 790 ; N apple ; B 56 -3 733 808 ;
  C 241 ; WX 329 ; N angleright ; B 21 -198 302 746 ;
  C 242 ; WX 274 ; N integral ; B 2 -107 291 916 ;
--- 177,192 ----
  C 227 ; WX 790 ; N copyrightsans ; B 49 -15 739 675 ;
  C 228 ; WX 786 ; N trademarksans ; B 5 293 725 673 ;
  C 229 ; WX 713 ; N summation ; B 14 -108 695 752 ;
! C 230 ; WX 384 ; N parenlefttp ; B 40 -290 436 926 ;
! C 231 ; WX 384 ; N parenleftex ; B 40 -80 92 920 ;
! C 232 ; WX 384 ; N parenleftbt ; B 40 -293 436 920 ;
! C 233 ; WX 384 ; N bracketlefttp ; B 0 -75 341 925 ;
! C 234 ; WX 384 ; N bracketleftex ; B 0 -75 55 925 ;
! C 235 ; WX 384 ; N bracketleftbt ; B 0 -75 340 925 ;
! C 236 ; WX 494 ; N bracelefttp ; B 201 -75 439 925 ;
! C 237 ; WX 494 ; N braceleftmid ; B 14 -75 255 925 ;
! C 238 ; WX 494 ; N braceleftbt ; B 201 -75 439 925 ;
! C 239 ; WX 494 ; N braceex ; B 201 -75 255 925 ;
  C 240 ; WX 790 ; N apple ; B 56 -3 733 808 ;
  C 241 ; WX 329 ; N angleright ; B 21 -198 302 746 ;
  C 242 ; WX 274 ; N integral ; B 2 -107 291 916 ;
***************
*** 196,206 ****
  C 246 ; WX 384 ; N parenrighttp ; B 54 -293 450 926 ;
  C 247 ; WX 384 ; N parenrightex ; B 398 -85 450 925 ;
  C 248 ; WX 384 ; N parenrightbt ; B 54 -293 450 926 ;
! C 249 ; WX 384 ; N bracketrighttp ; B 22 -80 360 926 ;
! C 250 ; WX 384 ; N bracketrightex ; B 305 -79 360 925 ;
! C 251 ; WX 384 ; N bracketrightbt ; B 20 -80 360 926 ;
! C 252 ; WX 494 ; N bracerighttp ; B 17 -75 255 926 ;
! C 253 ; WX 494 ; N bracerightmid ; B 201 -85 442 935 ;
! C 254 ; WX 494 ; N bracerightbt ; B 17 -70 255 926 ;
  EndCharMetrics
  EndFontMetrics
--- 196,209 ----
  C 246 ; WX 384 ; N parenrighttp ; B 54 -293 450 926 ;
  C 247 ; WX 384 ; N parenrightex ; B 398 -85 450 925 ;
  C 248 ; WX 384 ; N parenrightbt ; B 54 -293 450 926 ;
! C 249 ; WX 384 ; N bracketrighttp ; B 22 -75 360 925 ;
! C 250 ; WX 384 ; N bracketrightex ; B 305 -75 360 925 ;
! C 251 ; WX 384 ; N bracketrightbt ; B 20 -75 360 925 ;
! C 252 ; WX 494 ; N bracerighttp ; B 17 -75 255 925 ;
! C 253 ; WX 494 ; N bracerightmid ; B 201 -75 442 925 ;
! C 254 ; WX 494 ; N bracerightbt ; B 17 -75 255 925 ;
  EndCharMetrics
+ italicCorrection integral 67
+ leftItalicCorrection integral 52
+ subscriptCorrection integral -10
  EndFontMetrics
