*** 1.1	1993/10/23 09:59:12
--- npx.h	1993/10/24 17:37:00
***************
*** 73,83 ****
  struct	save87 {
  	struct	env87 sv_env;		/* floating point control/status */
  	struct	fpacc87	sv_ac[8];	/* accumulator contents, 0-7 */
- #ifndef dontdef
  	u_long	sv_ex_sw;	/* status word for last exception (was pad) */
  	u_long	sv_ex_tw;	/* tag word for last exception (was pad) */
! 	u_char	sv_pad[8 * 2 - 2 * 4];	/* bogus historical padding */
! #endif
  };
  
  /* Cyrix EMC memory - mapped coprocessor context switch information */
--- 73,81 ----
  struct	save87 {
  	struct	env87 sv_env;		/* floating point control/status */
  	struct	fpacc87	sv_ac[8];	/* accumulator contents, 0-7 */
  	u_long	sv_ex_sw;	/* status word for last exception (was pad) */
  	u_long	sv_ex_tw;	/* tag word for last exception (was pad) */
! 	u_char	sv_pad[60];	/* bogus historical padding */
  };
  
  /* Cyrix EMC memory - mapped coprocessor context switch information */
